src/bootstrap.config.js

```diff
@@ -0,0 +1,21 @@
+import fs from 'fs';
+import path from 'path';
+import yaml from 'js-yaml';
+import stripComments from 'strip-json-comments';
+
+import fileExists from './utils/fileExists';
+
+
+const CONFIG_FILE = '.bootstraprc';
```

Comment:
I'd rather put `bootstrapVersion: 4` in `.bootstraprc`, there is no use cases for multiple versions of this config in one project.